test/select.jl

```diff
@@ -1608,4 +1608,25 @@ end
     @test_throws ArgumentError select(df, :a => (x -> 1) => :a1, :a => (x -> [2]) => :a2, [:a])
 end
 
+@testset "normalize_selection" begin
+    @test DataFrames.normalize_selection(
+            DataFrames.Index(Dict(:a => 1, :b => 2), [:a, :b]),
+            [:a] => sum,
+            false) ==
+        (1 => (sum => :a))
```

Comment:
We don't use this style in DataFrames: rather keep the first argument on the first line, and align other arguments to it.